rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -1,7 +1,18 @@
 == Hardware Requirements
 
 * For optimal performance, use a dedicated computer for running the RPA Bot.
-
+* The following table shows the minimum and the recommended hardware requirements for running one process. To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
```

Comment:
Should this bullet be written more as a requirement with the details below?

* Use a computer with the required amount of RAM.
   The amount of RAM required depends on the number of processes you plan to run in parallel.